package.json

```diff
@@ -92,7 +92,7 @@
     "packages/perf-benchmarks"
   ],
   "engines": {
-    "node": ">=10.16.3 <10.17.0",
+    "node": ">=10.16.3 <13.0.0",
```

Comment:
I believe we want to peg this at `~10.16.3` because we want to run the same version as SFCI